pkg/controllers/rollout/controller.go

```diff
@@ -186,20 +188,17 @@ func pickBindingsToRoll(allBindings []*fleetv1beta1.ClusterResourceBinding, late
 	// Those are the bindings that are candidates to be updated to latest resources during the rolling phase.
 	updateCandidates := make([]*fleetv1beta1.ClusterResourceBinding, 0)
 
-	// fill out all the default values for CRP so we don't need to check nil values
-	// TODO: remove this after we have mutation webhook
-	fleetv1beta1.SetDefaultsClusterResourcePlacement(crp)
```

Comment:
why do we remove this?

Reply:
it's already called in the reconcile call